Marlin/src/HAL/HAL_ESP32/spiffs.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ * Copyright (c) 2016 Bob Cousins bobcousins42@googlemail.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef ARDUINO_ARCH_ESP32
+
+#include "../../inc/MarlinConfigPre.h"
+
+#if EITHER(WEBSUPPORT, EEPROM_SETTINGS)
+
+#include "../../core/serial.h"
+
+#include "FS.h"
+#include "SPIFFS.h"
+
+bool spiffs_initialized;
+
+void spiffs_init() {
+  if (SPIFFS.begin()) {
+    spiffs_initialized = true;
+  } else {
+    SERIAL_ECHO_MSG("SPIFFS Mount Failed");
```

Comment:
Could format it instead, but I chose to replicate the old behaviour from web.cpp, at least for now.